cms/event/digitalise-your-supply-chains-to-save-cost-improve-efficiency-overall-profitability-with-trames.md

```diff
@@ -1,12 +1,13 @@
 ---
 title: Digitalise Your Supply Chains to save cost, improve efficiency & Overall
   Profitability with Trames
+thumbnail: https://images.unsplash.com/photo-1551818255-e6e10975bc17?ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8Y29uZmVyZW5jZXxlbnwwfHwwfHw%3D&ixlib=rb-1.2.1&w=1000&q=80
 description: Designed for businesses of all sizes, learn how you can digitise
   your supply chains to save cost, improve efficiency and overall profitability
   by using the Trames platform.
 link: https://trames.sg/registration
-date: 8 Apr 2021
+date: 1 Dec 2021
 timeStart: 2021-03-19T07:00:00.000Z
```

Comment:
why date change?

Reply:
oh i wanted to show the register button design. ok i changed it back and created another dummy event for testing future events.

Reply:
maybe storybook?

Reply:
done.